sdks/python/apache_beam/transforms/sql.py

```diff
@@ -74,3 +77,8 @@ def __init__(self, query, dialect=None):
             SqlTransformSchema(query=query, dialect=dialect)),
         BeamJarExpansionService(
             ':sdks:java:extensions:sql:expansion-service:shadowJar'))
+
+
+class Row(object):
+  def __init__(self, **kwargs):
+    self.__dict__.update(kwargs)
```

Comment:
This is definitely useful for SQL but I don't think it should be SQL-specific. What about putting it in `apache_beam/typehints/schemas.py`?

Reply:
Hm yeah that makes sense. Maybe we need a new file or package for user-facing schema tools?

Reply:
Is there going to be any other top-level schema stuff that it would make sense for this to be a sibling to? Otherwise, perhaps I could just put at a top level, and one would use it as `apache_beam.Row`.

Reply:
Mm nothing comes to mind. I suppose it could just be `apache_beam.Row` for now, and we can alias it if we add a schema package with other top-level schema stuff later.

Reply:
Moved to pvalue (imported to the top level), like TaggedOutput.